.gitignore

```diff
@@ -25,6 +25,8 @@ build/Release
 
 node_modules/
 
+/coverage
```

Comment:
What is this folder for?

Reply:
Coverages from `jest`. We do not use it, but someone might want to run `jest --coverage`. For example, I did that. So this is more "just in case" .  The `/coverage` is used to match only the folder in project root, otherwise it also matches the `Coverage` inside `Sdk` folder.